generator/osm2meta.cpp

```diff
@@ -386,3 +386,89 @@ string MetadataTagProcessorImpl::ValidateAndFormat_duration(string const & v) co
   return format(hours);
 }
 
+
+MetadataTagProcessor::~MetadataTagProcessor()
+{
+  if (!m_description.IsEmpty())
+    m_params.GetMetadata().Set(feature::Metadata::FMD_DESCRIPTION, m_description.GetBuffer());
+}
+
+void MetadataTagProcessor::operator()(std::string const & k, std::string const & v)
+{
+  if (v.empty())
+    return;
+
+  using feature::Metadata;
+  Metadata & md = m_params.GetMetadata();
+
+  if (strings::StartsWith(k, "description"))
+  {
+    // Process description tags.
+    int8_t lang = StringUtf8Multilang::kDefaultCode;
+    size_t const i = k.find(':');
+    if (i != std::string::npos)
+    {
+      int8_t const l = StringUtf8Multilang::GetLangIndex(k.substr(i+1));
+      if (l != StringUtf8Multilang::kUnsupportedLanguageCode)
+        lang = l;
+    }
```

Comment:
Possibly worth to support `description:covid19` tag, it has 10981 uses worldwide https://taginfo.openstreetmap.org/keys/description:covid19
I think `lang` will be unchanged i.e. `StringUtf8Multilang::kDefaultCode`
I am afraid it might overwrite `description` value

Reply:
Well, looks like it worth in current situation. But now we can't add any additional language (see https://github.com/organicmaps/organicmaps/blob/master/coding/string_utf8_multilang.cpp#L81)

Probably, we can use some kReservedLang language for this ..